.github/workflows/ci.yml

```diff
@@ -238,6 +238,8 @@ jobs:
           set PATH=%CONDA_PREFIX%\Library\lib;%PATH%
           set CPATH=%CONDA_PREFIX%\include
           set TBB_DLL_PATH=%CONDA_PREFIX%\Library\bin
+          set CL_CONFIG_TBB_DLL_PATH=%CONDA_PREFIX%\Library\bin
+          powershell -command "(Get-Content %CONDA_PREFIX%\Library\lib\cl.cfg) -replace 'CL_CONFIG_TBB_DLL_PATH =', 'CL_CONFIG_TBB_DLL_PATH = %CONDA_PREFIX%\Library\bin' | Out-File -encoding ASCII %CONDA_PREFIX%\Library\lib\cl.cfg"
```

Comment:
Let's add a comment about the fix if yaml allows doing that. It is difficult for  another person to understand what is going on here.

Reply:
we can't add comments in the run: section. hence added prier to that.